Content/Skills/Melee/Berserk.cs

```diff
@@ -0,0 +1,29 @@
+﻿using FunnyExperience.Content.Buffs;
+using FunnyExperience.Content.Items.Gear;
+using FunnyExperience.Core.Systems.SkillSystem;
+
+namespace FunnyExperience.Content.Skills.Melee
+{
+	public class Berserk : Skill
+	{
+		public Berserk(int duration, int timer, int maxCooldown, int cooldown, int manaCost, GearType weaponType) : base(duration, timer, maxCooldown, cooldown, manaCost, weaponType)
```

Comment:
Keeping this constructor parameterized is likely a poor choice, if we make these parameterless we can use IoC to build a list of prototypes to get autoloading functionality and not need to manually initialize instances

Reply:
The way Skill.cs is currently written would require a parameterized constructor though